src/indra_cogex/indexing/cli.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+"""Run the database indexing."""
+from typing import Optional, Tuple
+
+import click
+
+
+@click.command(name="Build extra indexes")
```

Comment:
this name  might not do what you expect - this would get used if it's added as a subcommand in another CLI (spaces are an issue)

Reply:
Ah, beginner's mistake, will fix

Reply:
Resolved in https://github.com/bgyori/indra_cogex/pull/55/commits/89cdb5a00b64636b41e705992e6019c30c44262d